### PR TITLE
Separate formatting check

### PR DESCRIPTION
**Pull request by Shane32**
- Faster build checks
- Allows to see formatting problems separate from test failures
- Same as server repo

### CODE REVIEW

File: .github/workflows/format.yml
Comment (sungam3r) [inline .github/workflows/format.yml:22]:
```suggestion
      - name: Setup .NET SDK
```

**Comment by sungam3r** [comment]
https://github.com/graphql-dotnet/authorization/pull/254#discussion_r1064056053

OK, I'm fine with moving formatting into new workflow. Just want to clarify merge button behavior. Format check failed. Merge button is green. Right? Good behavior or not?

**Comment by Shane32** [comment]
> [graphql-dotnet/authorization#254 (comment)](https://github.com/graphql-dotnet/authorization/pull/254#discussion_r1064056053)
> 
> OK, I'm fine with moving formatting into new workflow. Just want to clarify merge button behavior. Format check failed. Merge button is green. Right? Good behavior or not?

Either way, I'll fix the formatting before merging.  As we are both admins of the repo, and can merge regardless of build checks, and as there are no other contributors currently, it makes little difference if there are merge restrictions.

**Comment by sungam3r** [comment]
I expected such an answer. OK. I'm done with reviews today. 3:20 AM.

**Review by sungam3r — APPROVED** [review]

**Pull request merged**
